src/components/calcite-tooltip-manager/calcite-tooltip-manager.e2e.ts

```diff
@@ -321,4 +321,39 @@ describe("calcite-tooltip-manager", () => {
 
     expect(await hoverTip.getProperty("open")).toBe(true);
   });
+
+  it("should close hovered tooltip when clicked", async () => {
+    const page = await newE2EPage();
```

Comment:
Can you pass the HTML as a parameter when the creating the page and use the `html` helper to automate formatting?

Reply:
Muchas thanks por updating las otras tests! 🥇